services/cosi/cosi.go

```diff
@@ -41,24 +40,30 @@ type SignatureRequest struct {
 
 // SignatureResponse is what the Cosi service will reply to clients.
 type SignatureResponse struct {
-	Sum       []byte
+	Hash      []byte
 	Signature []byte
+	Aggregate abstract.Point
```

Comment:
I put it in because I want the web-interface to be able to select any number of nodes and then request a signature using those. So instead of calculating the aggregate key in JavaScript, I calculate it in go and send it back. But yes, that's bad, I should learn how to calculate aggregate public keys in javascript...